wmn-data.json

```diff
@@ -2393,6 +2393,16 @@
         "known" : ["carlos01", "paul"],
         "cat" : "hobby"
        },
+       {
+        "name" : "Holopin",
+        "uri_check" : "https://holopin.io/@{account}#",
+        "e_code" : 200,
+        "e_string" : "Holobytes",

```

Comment:
Instead of making this a single word that could appear on a page with no profile, consider changing this to ` | Holopin</title>`